### DISCUSSION TITLE
GIF中都有些什么？ - 细节个人博客 - 一个正在默默努力的前端菜鸟的个人博客网站

### DISCUSSION THREAD

**Discussion by giscus** [humandetail/squad-blog-frontend#8] [Announcements]
# GIF中都有些什么？ - 细节个人博客 - 一个正在默默努力的前端菜鸟的个人博客网站

细节个人博客是一个记录学习笔记以及前端技术的原创独立的个人博客(Humandetail.com)

https://www.humandetail.com/posts/Jy8OR34zYM

**Comment by humandetail** [comment]
test

**Comment by libondev** [comment]
把钱退我我不玩了

**Reply by humandetail** [reply]
谢谢

**Comment by libondev** [comment]
# 快点更新, 已经一个季度不更新了, 什么驴敢这么歇啊!!!
